saleor/payment/gateways/adyen/plugin.py

```diff
@@ -105,7 +105,7 @@ class AdyenGatewayPlugin(BasePlugin):
             ),
             "label": "Origin Url",
         },
-        "Live": {
+        "live": {
             "type": ConfigurationTypeField.STRING,
             "help_text": (
                 "Leave it blank when you want to use test env. To communicate with the"
```

Comment:
What should I put here for a live environment?

Reply:
I've added here the link to docs - `https://docs.adyen.com/development-resources/live-endpoints`. line 116. Is it not enough?

Reply:
👍